### PR TITLE
Adding combobox and Checkbox

### PR DESCRIPTION
**Pull request by MadayAlcala**
Hi guys, I added the checkbooks and combobox needed for ConvertVideo and ConvertPDF, please check it out.

### CODE REVIEW

File: src/main/java/com/fundation/logic/view/convert/ConvertPDFPanel.java [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+        JLabel lblThumbnailFormat = new JLabel("Format: ");
+        GridBagConstraints gbc_lblThumbnailFormat = new GridBagConstraints();
+        gbc_lblThumbnailFormat.anchor = GridBagConstraints.EAST;
+        gbc_lblThumbnailFormat.insets = new Insets(0, 0, 5, 5);
+        gbc_lblThumbnailFormat.gridx = 2;
+        gbc_lblThumbnailFormat.gridy = 7;
+        add(lblThumbnailFormat, gbc_lblThumbnailFormat);
+
+        comboBoxThumbnailFormat = new JComboBox();
+        comboBoxThumbnailFormat.setModel(new DefaultComboBoxModel(new String[]{"bmp", "png", "jpg"}));
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/view/convert/ConvertPDFPanel.java]: All previous implemented comboBox show options in uppercase.

File: src/main/java/com/fundation/logic/view/convert/ConvertVideoPanel.java [hunk 2]
```diff
@@ -204,11 +206,28 @@ private void videoCriteriaConvertPanel() {
...
+        JLabel lblFormatKeyframe = new JLabel("Format:");
+        GridBagConstraints gbc_lblFormatKeyframe = new GridBagConstraints();
+        gbc_lblFormatKeyframe.anchor = GridBagConstraints.EAST;
+        gbc_lblFormatKeyframe.insets = new Insets(0, 0, 5, 5);
+        gbc_lblFormatKeyframe.gridx = 2;
+        gbc_lblFormatKeyframe.gridy = 13;
+        add(lblFormatKeyframe, gbc_lblFormatKeyframe);
+
+        comboBoxKeyFrameFormat = new JComboBox();
+        comboBoxKeyFrameFormat.setModel(new DefaultComboBoxModel(new String[]{"png", "jpg", "tiff"}));
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/view/convert/ConvertVideoPanel.java]: All previous implemented comboBox show options in uppercase.

File: src/main/java/com/fundation/logic/view/convert/ConvertVideoPanel.java [hunk 3]
```diff
@@ -245,11 +264,28 @@ private void videoCriteriaConvertPanel() {
...
+        JLabel lblThumbnailFormat = new JLabel("Format:");
+        GridBagConstraints gbc_ThumbnailFormat = new GridBagConstraints();
+        gbc_ThumbnailFormat.anchor = GridBagConstraints.EAST;
+        gbc_ThumbnailFormat.insets = new Insets(0, 0, 5, 5);
+        gbc_ThumbnailFormat.gridx = 2;
+        gbc_ThumbnailFormat.gridy = 14;
+        add(lblThumbnailFormat, gbc_ThumbnailFormat);
+
+        comboBoxThumbnailFormat = new JComboBox();
+        comboBoxThumbnailFormat.setModel(new DefaultComboBoxModel(new String[]{"bmp", "png", "jpg"}));
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/view/convert/ConvertVideoPanel.java]: All previous implemented comboBox show options in uppercase.

File: src/main/java/com/fundation/logic/view/convert/ConvertPDFPanel.java [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+    public void initComponent() {
+        settingPanel = new PanelSetter();
+        settingPanel.setPanel(this);
+        showPPTConvertCriteriaForm();
+    }
+
+    /**
+     * This method show convert PDF form.
+     */
+    private void showPPTConvertCriteriaForm() {
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/view/convert/ConvertPDFPanel.java]: Method name says PPT

File: src/main/java/com/fundation/logic/view/convert/ConvertPDFPanel.java (hunk 1 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/logic/view/convert/ConvertPDFPanel.java]: Done

File: src/main/java/com/fundation/logic/view/convert/ConvertVideoPanel.java (hunk 2 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/logic/view/convert/ConvertVideoPanel.java]: Solved.

File: src/main/java/com/fundation/logic/view/convert/ConvertPDFPanel.java (hunk 4 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/logic/view/convert/ConvertPDFPanel.java]: Solved.

File: src/main/java/com/fundation/logic/view/convert/ConvertVideoPanel.java (hunk 3 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/logic/view/convert/ConvertVideoPanel.java]: Solved.

**Pull request merged**